packages/gatsby/cache-dir/app.js

```diff
@@ -160,8 +158,6 @@ apiRunnerAsync(`onClientEntry`).then(() => {
     loader.loadPage(`/404.html`),
     loader.loadPage(window.location.pathname),
   ]).then(() => {
-    const preferDefault = m => (m && m.default) || m
-    const Root = preferDefault(require(`./root`))
```

Comment:
This was causing some weird module dependency chains and it wasn't exactly playing nice with HMR - hot updates for gatsby-browser modules were declined even tho I was accepting them on line 23 (for `./api-runner-browser`).

I'm not exactly sure why it was done this way (delayed commonjs import) :/